# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/ppc-inotify-init1.patch
# Copyright (C) 2015 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dietlibc-0.33/ppc/syscalls.h.vanilla	2015-10-28 13:33:27.190274378 +0000
+++ dietlibc-0.33/ppc/syscalls.h	2015-10-28 13:32:58.518345053 +0000
@@ -311,6 +311,7 @@
 #define __NR_subpage_prot	310
 #define __NR_timerfd_settime	311
 #define __NR_timerfd_gettime	312
+#define __NR_inotify_init1      318
 
 #define syscall_weak(name,wsym,sym) \
 .text; \
--- dietlibc-0.33/ppc64/syscalls.h.vanilla	2015-10-28 13:32:11.582460674 +0000
+++ dietlibc-0.33/ppc64/syscalls.h	2015-10-28 13:32:37.014398040 +0000
@@ -311,7 +311,7 @@
 #define __NR_subpage_prot	310
 #define __NR_timerfd_settime	311
 #define __NR_timerfd_gettime	312
-
+#define __NR_inotify_init1      318
 
 #define __diet_proto_common(sym) \
 	.section ".opd","aw"; \
